articles/machine-learning/how-to-authenticate-online-endpoint.md

```diff
@@ -42,7 +42,7 @@ To get the key or token, use [az ml online-endpoint get-credentials](/cli/azure/
 __Keys__ will be returned in the `primaryKey` and `secondaryKey` fields. The following example shows how to use the `--query` parameter to return only the primary key:
 
 ```azurecli
-ENDPOINT_CRED=$(az ml online-endpoint get-credentials -n $ENDPOINT_NAME -o tsv --query primaryKey)
+ENDPOINT_CRED=$(az ml online-endpoint get-credentials -n $ENDPOINT_NAME -g $RESOURCE_GROUP -w $WORKSPACE_NAME -o tsv --query primaryKey)
```

Comment:
This change is good. As I mentioned in the conversation, can you please update line 50 as well?